src/Sarif.Viewer.VisualStudio/TelemetryProvider.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.ApplicationInsights;
```

Comment:
>Microsoft.ApplicationInsights; [](start = 6, length = 30)

using System directives should come first. you might need to change your VS settings.

Reply:
Cool, didn't know about that option.